lm_eval/evaluator.py

```diff
@@ -215,7 +218,10 @@ def evaluate(
             ctx = task.fewshot_context(
                 doc=doc, num_fewshot=num_fewshot, rnd=rnd, description=description
```

Comment:
will the key-value pair string `description` be prepended to the few-shot context?

Reply:
confirmed that it is prepended:

![Screen Shot 2023-03-14 at 11 01 20 AM](https://user-images.githubusercontent.com/680489/225096911-d07c205c-b4e2-4532-bf01-34ae2c27e671.png)

Reply:
to support both cases, one option is to write a string of json in the config, e.g. `{"description": "...will be prepended to prompt...", "params": "majority_voting=4,sampling_temperature=0.3,eval_batch_size=4"}`, then use `json.loads` above this line